src/cli/cli.cpp

```diff
@@ -819,7 +819,6 @@ otError Interpreter::ProcessBackboneRouterMgmtMlr(Arg aArgs[])
      * @par
      * Returns the Multicast Listeners with the #otBackboneRouterMulticastListenerInfo
      * `mTimeout` in seconds.
-     * @note Only for reference and for testing the Backbone Router device.
```

Comment:
Shouldn't this have the "Available when `OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE` is enabled." note?